dist/docker/redhat/build_docker.sh

```diff
@@ -88,7 +88,7 @@ bcp LICENSE-ScyllaDB-Source-Available.md /licenses/
 
 run microdnf clean all
 run microdnf --setopt=tsflags=nodocs -y update
-run microdnf --setopt=tsflags=nodocs -y install hostname python3 python3-pip kmod
+run microdnf --setopt=tsflags=nodocs -y install hostname python3 python3-pip kmod procps-ng
```

Comment:
If you are already making the change, can you please sort the pacakge names alphabetically?

Reply:
Done